Mail/Views/Thread/Message/Calendar/CalendarChoiceButton.swift

```diff
@@ -43,20 +49,37 @@ struct CalendarChoiceButton: View {
         }
         .allowsHitTesting(!isSelected)
     }
+
+    private func sendReply() {
+        guard let messageUid else { return }
+
+        let oldChoice = selectedChoice
+        selectedChoice = choice
+        Task {
+            do {
+                try await mailboxManager.calendarReply(to: messageUid, reply: choice)
+                snackbarPresenter.show(message: MailResourcesStrings.Localizable.snackbarCalendarChoiceSent)
+            } catch {
+                selectedChoice = oldChoice
```

Comment:
Cool to use a Task to do work out of the main queue. 
But should we jump back to the main actor just to mutate the selectedChoice to be safe ?